app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/TabInCollectionViewHolder.kt

```diff
@@ -25,18 +28,19 @@ import mozilla.components.feature.tab.collections.Tab as ComponentTab
 class TabInCollectionViewHolder(
     val view: View,
     val interactor: CollectionInteractor,
+    private val differentLastItem: Boolean = false,
```

Comment:
Why do we need this new param if it seems like the only place we use it, we're setting it to true always?

Reply:
This was a preparation for a try to use the same ViewHolder for other lists (that to not have a different UI for the last item), not just to have a common layout resource. I can remove it for the time being if it is confusing at this moment.